### ISSUE TITLE
How is authentication handled?

### ISSUE THREAD

**Issue by timblaktu** [jprouty/mint-amazon-tagger#54]
Having spent many hours splitting and characterizing amazon transactions manually, I'm intrigued and planning to give this tool a try. My biggest question is how it handles authentication to both amazon and mint. The command line accepts your email addrs for each, and no password, and I'm assuming the tools asks you to enter the password each time, but I'd want to make sure that it is using sufficient encryption in the connection.. @jprouty can you speak to this (or perhaps add a paragraph about this in the README)?

p.s. also you have some admirers [over on bogleheads](https://www.bogleheads.org/forum/viewtopic.php?f=11&t=226589&p=4973969#p3717974). I directed them here for asking questions if they wanted to get your attention.

**Comment by jprouty** [comment]
Hi @timblaktu. I just updated the main README. Thanks for the pointer over at bogleheads; I posted there years ago and haven't been back :)

The was this program works is via selenium, which drives a Chromium browser on your behalf. The username and password are provided either on demand, upfront at cli, or in the *new* GUI tool. The username/password are then used in the Chromium browser, to log into mint/amazon. The auth/encryption used standard SSL/HTTPS. In unglorified terms, this app is basically a fancy scrapper, and uses [another project](https://github.com/mrooney/mintapi) that wraps the Mint scrapping logic. I'm happy to provide more details if you'd like.

Finally, the last piece worth mentioning; all of the code is open sourced here, which means all of the above can easily be audited/verified.
